accounts/templates/accounts/participant_detail.html

```diff
@@ -75,78 +76,78 @@ <h4 class="child-name"><em>{{child.given_name}}</em></h4>
                 <div class="col-xs-7">
                      {{ demographics.state|default:"No Response" }}
                 </div>
             </div>
             <div class="row">
                 <div class="col-xs-5">
-                    <b> Area description: </b>
+                    <b> {% trans "Area description" %}: </b>
                 </div>
                 <div class="col-xs-7">
                     {{ demographics.density|default:"No Response" }}
                 </div>
             </div>
             <div class="row">
                 <div class="col-xs-5">
-                    <b> Languages Spoken at Home: </b>
+                    <b> {% trans "Languages Spoken at Home" %}: </b>
                 </div>
                 <div class="col-xs-7">
                     {{ demographics.languages_spoken_at_home|default:"No Response" }}
                 </div>
             </div>
             <div class="row">
                 <div class="col-xs-5">
-                    <b> Number of Children: </b>
+                    <b> {% trans "Number of Children" %}: </b>
                 </div>
                 <div class="col-xs-7">
                      {{ demographics.number_of_children|default:"No Response" }}
                 </div>
             </div>
             <div class="row">
                 <div class="col-xs-5">
-                    <b>  Children current ages: </b>
+                    <b> {% trans "Children current ages" %}: </b>
                 </div>
                 <div class="col-xs-7">
-                    {% for birthday in user.latest_demographics.child_birthdays %}{{birthday | timesince|default:"N/A"}}{% if not forloop.last %}; {% endif %}{% empty %}No Response{% endfor %}
+                    {% for birthday in user.latest_demographics.child_birthdays %}{{birthday | timesince|default:"N/A"}}{% if not forloop.last %}; {% endif %}{% empty %}{% trans No Response %}{% endfor %}
                 </div>
             </div>
             <div class="row">
                 <div class="col-xs-5">
-                    <b> Number of Guardians: </b>
+                    <b> {% trans "Number of Guardians" %}: </b>
                 </div>
                 <div class="col-xs-7">
-                    {{ demographics.number_of_guardians|default:"No Response" }}
+                    {{ demographics.number_of_guardians|default:"No Response"}}
                 </div>
             </div>
             <div class="row">
                 <div class="col-xs-5">
-                    <b> Explanation for Guardians: </b>
+                    <b> {% trans "Explanation for Guardians:" %} </b>
                 </div>
                 <div class="col-xs-7">
                     {{ demographics.number_of_guardians_explanation|default:"No Response" }}
                 </div>
             </div>
             <div class="row">
                 <div class="col-xs-5">
-                    <b> Race: </b>
+                    <b>{% trans "Race" %}:</b>
```

Comment:
will this sort of thing need a contextual marker or be fine?

Reply:
It does seem like a candidate, I'll highlight this general issue (with this example) to the translation group, and perhaps we will probably need to bring in contextual markers where needed.